doc/lua_api.txt

```diff
@@ -1964,6 +1964,23 @@ Elements
 * End of a container, following elements are no longer relative to this
   container.
 
+### `scroll_container[<X>,<Y>;<W>,<H>;<scrollbar name>;<direction>;<multiplier>]`
+
+* Start of a scroll_container block, all elements in a scroll_container block are
+  * moved by `(X, Y),
+  * additionally moved by the current value of the scrollbar with the name
+    `scrollbar name` times `multiplier` in the direction `direction` and
+  * clipped to the rectangle defined by `X`, `Y`, `W` and `H`.
+* Possible values for `direction` are `down` and `right`.
+* `multiplier` is optional and defaults to `0.1`.
+* Nesting is possible.
+* Some elements might work a little different if they are in a scroll_container.
```

Comment:
What would you like to say here? Different scaling?

Reply:
For example `bgcolor[color]` doesn't set the background of the whole formspec window but only of the scroll container. Also some elements that normally are never clipped are clipped in scroll_containers. Everything is intuitive, however, some people might get confused if they don't look whether the element is inside a scroll_container and expect the same to happen.